ComplementaryData/physiology/biomassComposition_Cofactor_Ion.tsv

```diff
@@ -0,0 +1,20 @@
+s_3880[c]	Ca(2+) 	2.17E-04	40.08	ion	DOI: 10.22161/ijeab/2.2.10
+s_3778[c]	chloride 	2.55E-03	35.45	ion	DOI: 10.22161/ijeab/2.2.11
+s_4019[c]	Cu2(+) 	6.59E-04	63.55	ion	DOI: 10.22161/ijeab/2.2.12
+s_0924[c]	iron(2+) 	3.04E-05	55.84	ion	DOI: 10.22161/ijeab/2.2.13
+s_3801[c]	Mn(2+) 	2.73E-03	54.94	ion	DOI: 10.22161/ijeab/2.2.14
+s_3822[c]	Zn(2+) 	7.48E-04	65.39	ion	DOI: 10.22161/ijeab/2.2.15
+s_1373[c]	potassium 	3.63E-03	39.1	ion	DOI: 10.22161/ijeab/2.2.16
+s_1437[c]	sodium 	3.97E-03	22.99	ion	DOI: 10.22161/ijeab/2.2.17
+s_4013[c]	Mg(2+) 	0.001242543	24.3	ion	DOI: 10.22161/ijeab/2.2.18

```

Comment:
all of the 9 above mentioned references seem to be incorrect, for example https://www.doi.org/10.22161/ijeab/2.2.10 is not a yeast paper

Reply:
Corrected.

Reply:
Is it from table 4? I do not see the composition for `chloride` in that table.